src/patternfly/components/Tabs/tabs.scss

```diff
@@ -2,17 +2,33 @@ $pf-c-tabs--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl", "2xl"
 $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
 .pf-c-tabs {
-  --pf-c-tabs--Inset: 0;
-  --pf-c-tabs--before--BorderStyle: solid;
+  --pf-c-tabs--inset: 0;
+  --pf-c-tabs--xl--inset: var(--pf-global--spacer--sm);
   --pf-c-tabs--before--BorderColor: var(--pf-global--BorderColor--100);
-  --pf-c-tabs--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs--before--border-width--base: var(--pf-global--BorderWidth--sm);
   --pf-c-tabs--before--BorderTopWidth: 0;
   --pf-c-tabs--before--BorderRightWidth: 0;
-  --pf-c-tabs--before--BorderBottomWidth: var(--pf-c-tabs--before--BorderWidth);
+  --pf-c-tabs--before--BorderBottomWidth: var(--pf-c-tabs--before--border-width--base);
   --pf-c-tabs--before--BorderLeftWidth: 0;
-  --pf-c-tabs--m-vertical--Inset: var(--pf-global--spacer--lg);
+  --pf-c-tabs--Transition: padding .125s;
```

Comment:
Should you split this up to be `--pf-c-tabs--TransitionProperty` `--pf-c-tabs--TransitionDuration`